### DISCUSSION TITLE
3D Multi-organ Segmentation with Swin UNETR

### DISCUSSION THREAD

**Discussion by anuragp2018** [Project-MONAI/tutorials#1642] [Q&A]
Please help me to run this model on custom data set in 3D slicer.

**Comment by diazandr3s** [comment]
Hi @anuragp2018,

For 3DSlicer integration, have you tried using MONAI Label for this?

Here are the instructions: https://github.com/Project-MONAI/MONAILabel?tab=readme-ov-file#getting-started-with-monai-label

You could start using the Radiology app with the Segmentation model: https://github.com/Project-MONAI/MONAILabel/tree/main/sample-apps/radiology#pretrained-models

Video tutorials: https://youtu.be/8y1OBQs2wis?si=a_0pW3u5TQpx0Hyo

Let us know,
